### ISSUE TITLE
Lemoji contributions guide

### ISSUE THREAD

**Issue by gustavwilliam** [python-discord/branding#104]
Lemojis are great and I've heard about many people being interested in contributing. There are however many steps involved and things to know before getting started, which is why I'm proposing a lemoji contributions guide. It will mention at least the following core points:
- The lemojis are based on Twemojis
- How relevant Twemojis are found and downloaded
- We work with vector graphics, which means that the contributors should be based on the source files (svgs)
- What lemoji template they should use
- Our workflow and possibly links to the contributions guides
- Some general advice

If we keep it clear and concise, I think this short guide (located in the `emotes/lemojis/` folder) will help keep consistent and high quality contributions. We will still talk to all contributors who are interested in being part of the process and mention the importance of our communication in `#dev-branding`. This guide will simply aid in making sure the contributions are of the highest possible quality.

**Comment by lemonsaurus** [comment]
So basically just a README in the lemoji folder, right?

I'm not mad at that.

**Comment by gustavwilliam** [comment]
Yes, exactly like that. Would we be able to call it `CONTRIBUTING.md` and have it displayed automatically when going into the directory, or would we need to call it `README.md`? I think both would be fine.

**Comment by lemonsaurus** [comment]
I think it has to be a readme to be displayed, but not sure.

**Comment by Akarys42** [comment]
It has to be called `README`, yes. The extension doesn't matter.

**Comment by lemonsaurus** [comment]
I'd still very much like to see this one happen.

**Comment by lemonsaurus** [comment]
Whoops, lol.
